src/abi/linux_abi.rs

```diff
@@ -640,12 +640,22 @@ pub enum Opcode {
     CopyFileRange = 47,
     SetupMapping = 48,
     RemoveMapping = 49,
+    MaxOpcode = 50,
 
     /* Reserved opcodes: helpful to detect structure endian-ness in case of e.g. virtiofs */
     CuseInitBswapReserved = 1_048_576, /* CUSE_INIT << 8 */
     InitBswapReserved = 436_207_616,   /* FUSE_INIT << 24 */
 }
 
+impl Opcode {
```

Comment:
Could we use
impl From<u32> for Opcode {
}
?

Reply:
sounds good!

Reply:
fixed